DesignSafe-CI/portal
# Task/des 2513 referenced data text changes

## Overview: ##
Changes to the text: Referenced Data

## PR Status: ##


## Related Jira tickets: ##

* [DES-2513](https://jira.tacc.utexas.edu/browse/DES-2513)

## Summary of Changes: ##
Changed Referenced Data to Referenced Data and Software
Changed Add another Referenced Data to Add Another Referenced Data/Software
Changed Remove Referenced Data to Remove Referenced Data/Software

## Testing Steps: ##
My Projects
1. When creating a new project, the new text should appear in the form
2. In the curation model, the new text should appear.
3. When publishing, in the Proofread {ProjType}, when you click Edit, new text should appear.
4. When amending a project, click Edit {ProjType} Metadata, new text should appear.

Published
1. Projects that have a ref data should now show Referenced Data and Software

## UI Photos:
![Screen Shot 2023-07-07 at 1 52 33 PM](https://github.com/DesignSafe-CI/portal/assets/35277477/d8366057-c7f8-4cdb-8b8c-a4ebdf86b891)
![Screen Shot 2023-07-07 at 1 49 20 PM](https://github.com/DesignSafe-CI/portal/assets/35277477/28549ae0-a4a9-4cd3-9cac-e37d585c5eef)

## Notes: ##
On the Edit/Add {ProjType} page, I didn't change "URL or DO for the referenced data, in URL format" because that would have made it two lines, and it didn't look good. 

Also, I made this it's own ticket because of the large amount of file changes.
